test/run_tests.pl

```diff
@@ -62,7 +62,9 @@ BEGIN
     @tests = map { abs2rel($_, rel2abs(curdir())); } @tests;
 
     my $harness = $TAP_Harness->new(\%tapargs);
-    $harness->runtests(sort @tests);
+    my $ret = $harness->runtests(sort @tests);
+
+    exit $ret->has_errors if (ref($ret) eq "TAP::Parser::Aggregator");
```

Comment:
Just in case it wasn't clear, modification was originally tested and verified to work with TAP::Harness *and* without.